doc/release-notes/0.8/0.8.md

```diff
@@ -0,0 +1,76 @@
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.8 release notes
+
+## Supported platforms
+
+OpenJDK binaries that contain the Eclipse OpenJ9 virtual machine are available from the
+[AdoptOpenJDK community](https://adoptopenjdk.net/?variant=openjdk8-openj9). These builds are
+being tested against the OpenJDK test suite.
+
+The following platforms are available:
+
+|Operating system | Platform architecture | Comments                                    |
+|-----------------|-----------------------|---------------------------------------------|
+|Linux            | x86, IBM Power, IBM Z | Windows subsytem for Linux is not supported |
+|AIX              | IBM Power             | Untested                                    |
+|Windows          | x86                   | Untested                                    |
+
+
+## Known Issues
+
+The v0.8 release contains the following known issues and limitations:
+
+#### Direct dump reader
+[Issue #378](https://github.com/eclipse/openj9/issues/378):
+The direct dump reader is currently not working, which has an impact on diagnosing problems with the VM, Garbage Collector, and the JIT. For example, [issue #968](https://github.com/eclipse/openj9/issues/968).
+
+#### Signal handling
+[Issue #54](https://github.com/eclipse/openj9/issues/54): Some signals are not being handled correctly by the VM. In particular, for CTRL-C or other signals that shutdown the VM, the shutdown hooks are not running. See [Issue #378](https://github.com/eclipse/openj9/issues/378)
+
+#### Large Java heaps
+[Issue #479](https://github.com/eclipse/openj9/issues/479): The binaries available at AdoptOpenJDK are built only with the compressed references VM, which does not support Java heaps larger
+than 57 GB (or ~62 GB with command line options). When builds that contain the non-compressed references VM are available, this limitation will be removed. If
+you want to build a non-compressed references VM yourself, you can do so now by following our [detailed build instructions](https://github.com/eclipse/openj9/blob/master/buildenv/Build_Instructions_V8.md).
+
+#### Graphics Processor Unit (GPU) support
+GPU acceleration with `java.util.Arrays.sort()` is not supported, although `com.ibm.gpu.Maths` can be used directly.
+
+#### Functional verification tests
```

Comment:
I'm wondering if we should have this section. More issues have been reported since this list was created. Not necessarily from the functional verification test bucket, but from additional testing being running. It might make more sense to provide a link like https://github.com/eclipse/openj9/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22test+failure%22+-label%3Ajdk9+-label%3Ajdk10+, although this link is only good at the time when we branch for the 0.8 release.

Reply:
Reviewing now, thanks for the 2nd note, I somehow missed the first request on this one.

Reply:
I think we need a static list so that there is a fixed reference point for test failures. I've made the release notes match the query above. We'll have to do a last minute review to ensure that we still line up at GA.